src/lib/libcore.js

```diff
@@ -1417,11 +1417,18 @@ addToLibrary({
     }
   },
 
-  emscripten_log__deps: ['$formatString', '$emscriptenLog'],
+  emscripten_log__deps: ['$stackAlloc', 'vasprintf', '$emscriptenLog', 'free'],
   emscripten_log: (flags, format, varargs) => {
```

Comment:
I'm not even sure how this works as written because `emscripten_log` doesn't take `va_list` but `...` so I would expect to see a `va_start` here to injest the `...` args.  I guess `va_start` on wasm is a no-op, or this wouldn't work?

Reply:
I might be misunderstanding it, but AFAIK va_start is just a macro that allows you to get a pointer to otherwise unnamed `...` varargs binding?